schemas/src/main/java/com/scottlogic/deg/schemas/common/ValidationResult.java

```diff
@@ -5,9 +5,9 @@
 public class ValidationResult {
     public final List<String> errorMessages;
 
-    public ValidationResult(List<String> errorMessages){
-        this.errorMessages = errorMessages;
-    }
+    public ValidationResult(List<String> errorMessages) {this.errorMessages = errorMessages;}
+
+    public void addErrorMessages(List<String> errorMessages) {this.errorMessages.addAll(errorMessages);}
```

Comment:
Minor comment, content should be on a new line

Reply:
yep. reformatted